<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
				background: url(:/images/images/background.png);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_alarm{	
	border-radius:15px;
}
QPushButton#btn_calendar{	
	border-radius:15px;
}
QPushButton#btn_weather{	
	border-radius:15px;
}
QPushButton#btn_notes{	
	border-radius:15px;
}
QPushButton#btn_swatch{	
	border-radius:15px;
}

</string>
    </property>
    <widget class="QLabel" name="lbl_clock">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>43</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; color:#039ea1;&quot;&gt;15 : 15&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_alarm">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/images/alarm.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_calendar">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/images/calendar.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_weather">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>200</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/images/weather.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_notes">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>200</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/images/notes.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_swatch">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>200</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/images/stopwatch.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_quit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>550</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_quit	{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#btn_quit:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 226));
}
QPushButton#btn_quit:pressed{	
	padding-left:3px;
	padding-top:3px;
	background-color:rgba(105, 118, 132, 200);
}</string>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
